src/TTCache.php

```diff
@@ -32,6 +32,8 @@ class TTCache
 
     protected TaggedStore $cache;
 
+    protected ?Result $lastResult;
```

Comment:
Knew we'd have to add something for telemetry at some point, but I don't think that's the right approach here. Can we remove telemetry related updates from this PR to get the `loaded / missing` keys update finalized on its own first.

Reply:
Sounds good 👍